packages/twenty-front/src/modules/users/components/UserProvider.tsx

```diff
@@ -14,6 +17,16 @@ export const UserProvider = ({ children }: React.PropsWithChildren) => {
 
   const dateTimeFormat = useRecoilValue(dateTimeFormatState);
 
+  const currentWorkspaceMember = useRecoilValue(currentWorkspaceMemberState);
+
+  const [locale, setLocale] = useState<Locale | undefined>();
+
+  useEffect(() => {
```

Comment:
Maybe we can get rid of that useEffect and load the date locales where we load the other locals already? See "dynamicActivate" (maybe update I18nFrontDecorator too)?

Reply:
Yes